src/ogg/flac/properties.rs

```diff
@@ -32,6 +32,7 @@ where
 	let info = stream_info.read_u32::<BigEndian>()?;
 
 	let sample_rate = info >> 12;
+	let bits_per_sample = ((info >> 4) & 0b11111) + 1;
 	let channels = ((info >> 9) & 7) + 1;
```

Comment:
Here is also one case where binary format is more readable.

Reply:
Yeah this one's fine. I was using decimal for the channels anyway.